core/enumerable/uniq_spec.rb

```diff
@@ -0,0 +1,21 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+require File.expand_path('../fixtures/classes', __FILE__)
+
+ruby_version_is '2.4' do
+  describe 'Enumerable#uniq'do
+    before :each do
+      @enum = Object.new.to_enum
+      class << @enum
+        def each
+          yield 0, 'foo'
+          yield 1, 'FOO'
+          yield 2, 'bar'
+        end
+      end
+    end
+
+    it 'returns an array that contains only unique result of the given block' do
```

Comment:
Thank you! Updated as it in https://github.com/ruby/spec/pull/447/commits/2fa99e3b7f8498af228b861afe62f487fb3df080 🙇